scipy/stats/_distribution_infrastructure.py

```diff
@@ -5219,3 +5217,149 @@ def __rtruediv__(self, other):
 
     def __neg__(self):
         return self * -1
+
+
+class IMFTransformedDistribution(TransformedDistribution):
```

Comment:
If we think about it, `ShiftedScaleDistribution` is a special case of what is now called `IMFTransformedDistribution`. Not something for now, but just to think about for the future. To reduce duplication, should `ShiftedScaledDistribution` inherit from `IMFTransformedDistribution`, providing an override for `__init__` and any methods where knowing we're dealing with a linear transform let's us do better than the generic methods?

Reply:
Maybe. `ShiftedScaledDistribution` also allows a negative scale, though (so we don't have to have separate right and left versions of distributions like `weibull_min`/`weibull_max` and `gumbel_l`/`gumbel_r`), so it is not strictly a monotonic transformation.

Reply:
Exactly, `increasing` changes depending on the sign of the scale. It seems like the principled way to do things here, but obviously shouldn't hold up anything.

Reply:
Oh right, and then things start to look more complicated.